spec/unit/device_spec.lua

```diff
@@ -273,15 +273,15 @@ describe("device module", function()
 
             mock_ffi_input = require('ffi/input')
             stub(mock_ffi_input, "waitForEvent")
-            mock_ffi_input.waitForEvent.returns(true, {
+            mock_ffi_input.waitForEvent.returns(true, { {
```

Comment:
@NiLuJe https://github.com/koreader/koreader/commit/2a2d2baf739dd5bb89078c798c7474719849b311#commitcomment-49028640
> Meaning you prefer the "with a break" variant?

I think so, less for the one liner as for situations like these.